@xen-orchestra/lite/src/components/tasks/TasksTable.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <UiTable class="tasks-table">
+    <template #header>
+      <th>{{ $t("task.name") }}</th>
+      <th>{{ $t("task.object") }}</th>
+      <th>{{ $t("task.progress") }}</th>
+      <th>{{ $t("task.started") }}</th>
+      <th>{{ $t("task.estimated-end") }}</th>
```

Comment:
Missing icons.
If it is voluntary, it would be necessary to add a comment on FIGMA to ask Clémence to remove them too.